packages/terraform-modules-public/testnet/modules/validator/startup.sh

```diff
@@ -0,0 +1,193 @@
+#!/bin/bash
+
+# ---- Set Up Logging ----
+
+curl -sSO https://dl.google.com/cloudagents/install-logging-agent.sh
+bash install-logging-agent.sh
+
+# ---- Set Up Persistent Disk ----
+
+# gives a path similar to `/dev/sdb`
+DISK_PATH=`readlink -f /dev/disk/by-id/google-${attached_disk_name}`
+DATA_DIR=/root/.celo
+
+echo "Setting up persistent disk ${attached_disk_name} at $DISK_PATH..."
+
+DISK_FORMAT=ext4
+CURRENT_DISK_FORMAT=`lsblk -i -n -o fstype $DISK_PATH`
+
+echo "Checking if disk $DISK_PATH format $CURRENT_DISK_FORMAT matches desired $DISK_FORMAT..."
+
+# If the disk has already been formatted previously (this will happen
+# if this instance has been recreated with the same disk), we skip formatting
+if [[ $CURRENT_DISK_FORMAT == $DISK_FORMAT ]]; then
+  echo "Disk $DISK_PATH is correctly formatted as $DISK_FORMAT"
+else
+  echo "Disk $DISK_PATH is not formatted correctly, formatting as $DISK_FORMAT..."
+  mkfs.ext4 -m 0 -F -E lazy_itable_init=0,lazy_journal_init=0,discard $DISK_PATH
+fi
+
+echo "Mounting $DISK_PATH onto $DATA_DIR"
+mkdir -p $DATA_DIR
+mount -o discard,defaults $DISK_PATH $DATA_DIR
+[[ ${reset_geth_data} == "true" ]] && rm -rf $DATA_DIR/geth
+mkdir -p $DATA_DIR/account
+
+# ---- Install Docker ----
+
+echo "Installing Docker..."
+apt update && apt upgrade
```

Comment:
worth tossing in `-y`s here?  `apt` _can_ prompt, but maybe it does something smart depending on stdin and avoids prompting. regardless, it might be useful for development (_e.g._, avoid having the script prompt when you're running it in an ssh session)

Reply:
Probably it's worth. I can imagine for now it has not caused any issue without because the update does not need it AFAIK and for now neither the upgrade command (I think it ask for confirmation when it needs to upgrade/install new packages or remove some installed package because the upgrade). Anyway it does not hurt so I will add it.
Thanks